### PR TITLE
[Doc clarification] Update recommended place to store assets

### PR DESCRIPTION
**Pull request by green-arrow**
Addresses #1365 

Let me know if there's something you think needs any further clarification :)

### CODE REVIEW

File: source/localizable/addons-and-dependencies/managing-dependencies.md [hunk 1]
```diff
@@ -27,7 +27,9 @@ Thus it reloads your app if you install new dependencies via `bower install <dep
 
 ## Other assets
 
-Assets not available as an addon or Bower package should be placed in the `vendor` folder in your project.
+Third-party assets not available as an addon or Bower package should be placed in the `vendor` folder in your project.
```
Comment (tomdale) [inline source/localizable/addons-and-dependencies/managing-dependencies.md]: I think that this should say "Third-party JavaScript," not "Third-party assets." Are there any cases where someone would be using third-party non-JavaScript files that wouldn't be better served by being placed in `public/`?

File: source/localizable/addons-and-dependencies/managing-dependencies.md [hunk 2]
```diff
@@ -109,7 +111,11 @@ All style assets added this way will be concatenated and output as `/assets/vend
 
 ### Other Assets
 
-All other assets like images or fonts can also be added via `import()`.
+All assets located in the `public` folder will be copied as is to the `dist/` folder.
```
Comment (tomdale) [inline source/localizable/addons-and-dependencies/managing-dependencies.md]: Even though it's covered on this page, a new user skimming the documentation may not be clear about what the `dist/` folder is. Maybe this should mention that `dist/` is the final output directory. Also, for consistency, we should probably add a trailing slash to `public/`.

File: source/localizable/addons-and-dependencies/managing-dependencies.md [hunk 3]
```diff
@@ -109,7 +111,11 @@ All style assets added this way will be concatenated and output as `/assets/vend
...
+
+For example, a `favicon` located at `public/images/favicon.ico` will be copied to `dist/images/favicon.ico`.
+
+All other third-party assets like images or fonts can also be added via `import()`.
```
Comment (tomdale) [inline source/localizable/addons-and-dependencies/managing-dependencies.md]: This reads strangely to me because it says "all other third-party assets" comes after a discussion of assets that are explicitly _not_ third party. What does the other mean in this case?

It's not clear here whether this documentation is about files in `vendor/` or `public/`. Files in `public/` are included automatically, as you note. Files in `vendor/` **must** be `app.import()`ed. They won't work at all if you skip this step. We may want to state this in stronger language, and make it clear which rules apply to which directories.

TLDR:

1. Your own, non-JavaScript assets go in `public/` and are copied to the final output directory automatically. You should not (and indeed, cannot) `app.import()` anything in `public/`.
2. Third-party assets, _including_ JavaScript, go inside the `vendor/` directory. In order for a file to be included in the final build output, it _must_ be `app.import()`ed.

File: source/localizable/addons-and-dependencies/managing-dependencies.md (hunk 1 above)
Comment (green-arrow) [inline source/localizable/addons-and-dependencies/managing-dependencies.md]: I've tried to think of a scenario where a non-bower / non-npm third-party asset might contain assets other than JS, but the only thing I can think of would be if they're using a font that's technically from a third-party. Even in that case though, I think the asset would be better served in `public`.

File: source/localizable/addons-and-dependencies/managing-dependencies.md [hunk 4]
```diff
@@ -27,7 +27,9 @@ Thus it reloads your app if you install new dependencies via `bower install <dep
 
 ## Other assets
 
-Assets not available as an addon or Bower package should be placed in the `vendor` folder in your project.
+Third-party JavaScript not available as an addon or Bower package should be placed in the `vendor/` folder in your project.
```
Comment (green-arrow) [inline source/localizable/addons-and-dependencies/managing-dependencies.md]: Updated to 'Third-party JavaScript' instead of 'Third-party assets'.

File: source/localizable/addons-and-dependencies/managing-dependencies.md [hunk 5]
```diff
@@ -109,9 +111,15 @@ All style assets added this way will be concatenated and output as `/assets/vend
 
 ### Other Assets
 
-All other assets like images or fonts can also be added via `import()`.
-By default, they
-will be copied to `dist/` as they are.
+All assets located in the `public/` folder will be copied as is to the final output directory, `dist/`.
```
Comment (green-arrow) [inline source/localizable/addons-and-dependencies/managing-dependencies.md]: Updated to define exactly what `dist/` is (final output directory).

File: source/localizable/addons-and-dependencies/managing-dependencies.md [hunk 6]
```diff
@@ -109,9 +111,15 @@ All style assets added this way will be concatenated and output as `/assets/vend
...
+
+For example, a `favicon` located at `public/images/favicon.ico` will be copied to `dist/images/favicon.ico`.
+
+All third-party assets, included either manually in `vendor/` or via a package manager like Bower, must be added via `import()`.
```
Comment (green-arrow) [inline source/localizable/addons-and-dependencies/managing-dependencies.md]: Updated phrasing to more strongly state that third-party assets need to be included via `import`, otherwise they won't make the build.

File: source/localizable/addons-and-dependencies/managing-dependencies.md [hunk 7]
```diff
@@ -109,9 +111,15 @@ All style assets added this way will be concatenated and output as `/assets/vend
...
+
+Third-party assets that are not added via `import()` will not be present in the final build.
```
Comment (green-arrow) [inline source/localizable/addons-and-dependencies/managing-dependencies.md]: I think this may need a clarification that addons / NPM modules don't need the `import`, and this is only for things like bower packages and stuff in `vendor`. The previous sentence may be enough context, but I'm not sure. Thoughts?

**Comment by tomdale** [comment]
@green-arrow Awesome improvements, thank you!

**Comment by tomdale** [comment]
@emberjs/learning-team-managers LGTM, WDYT?

**Comment by green-arrow** [comment]
@tomdale thanks for the feedback!

**Comment by locks** [comment]
👌 

@homu r+
